Tests/Controller/HelperControllerTest.php

```diff
@@ -328,6 +329,20 @@ public function testappendFormFieldElementAction($validatorInterface)
 
         $twig = new \Twig_Environment($this->getMock('\Twig_LoaderInterface'));
         $twig->addExtension(new FormExtension($mockRenderer));
+
+        if (Kernel::MAJOR_VERSION >= 3 && Kernel::MINOR_VERSION >= 2) {
```

Comment:
Based on [this changelog](https://github.com/symfony/twig-bridge/blob/master/CHANGELOG.md#320), [here is the detection trick I'd use](https://github.com/sonata-project/SonataCoreBundle/pull/364/files#diff-ab34013c1c5e23c243a5c45874748062R61)